src/mixins/_icons.scss

```diff
@@ -6,8 +6,6 @@
   @include fd-icon-base($position);
   @include fd-icon-glyph($key, $position);
 
-  font-family: "SAP-icons";
```

Comment:
why do you remove it?

Reply:
Take a look how this `@mixin` is used:
```
some-button {
   @include fd-icon('slim-arrow-right') {
      font-size: 1rem;
   }
}
```

Previous implementation was adding `font-family: "SAP-icons"` to both: `.some-button` and `.some-button::before`.
I don't think that modifying styling of `.some-button` is a valid thing because icon exists only on its pseudo-elements.